tests/actor/test_config_manager.py

```diff
@@ -80,9 +80,7 @@ async def test_update(self, config_file: pathlib.Path) -> None:
         config_channel: Broadcast[Config] = Broadcast(
             "Config Channel", resend_latest=True
         )
-        _config_manager = ConfigManagingActor(
-            conf_file=str(config_file), output=config_channel.new_sender()
-        )
+        _config_manager = ConfigManagingActor(config_file, config_channel.new_sender())
```

Comment:
the parameter names were deleted, `config_file`  and `output`. Was this deliberately done? Just checking...

Reply:
Yes, I was hoping it passed unnoticed, damn you Dani! 😁

In a following commit I'm changing one of keyword arguments and since the original order in which I did the changes don't match the current commits it was more work to revise and do this in steps, so I thought it was fine to just remove the keywords from the call site to avoid breakage in the following commit. I even thought about making the arguments positional-only (as it is pretty clear what each argument is from the type alone) to avoid future breakages in general if we want to rename keyword args.

Reply:
Oh, I need to add a release note because this is a breaking change.

Reply:
Done!

Reply:
ok, then we only need an entry into the release notes here

Reply:
It was already added :)